### PR TITLE
 [0.5.0] add pipenv and readme to simulator 

### PR DESCRIPTION
**Pull request by mprevide**

### CODE REVIEW

File: client/README.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+Enter in the virtual environment:
+```shell
+pipenv shell
+```
+
+Check the [pipenv repository](https://github.com/pypa/pipenv/) for more commands and details.
+
+### Simulating a device
+
+#### Create a device to use with simulator
```
Comment (eduardogmisiuk) [inline client/README.md:35]: with the

File: client/README.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+```shell
+python3 create_lwm2m_devices.py http://localhost:8000  admin admin
+```
+
+This command will create n device (`unsecure-dev-0`) with 3 templates (`lwm2m`, `firmware_update` and `Temperature`) in a dojot running at
```
Comment (eduardogmisiuk) [inline client/README.md:41]: n devices

File: client/README.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+mvn package
+```
+
+To execute the simulator:
+
+```shell
+java -jar target/leshan-client-demo-1.0.0-M10-jar-with-dependencies.jar -n unsecure-client-endpoint-0 -u localhost:5683
+```
+
+In this case, the simulator will publish to device `unsecure-client-endpoint-0` in a dojot ruining at `localhost` to the `coap` port  (`udp`) `5683`.
```
Comment (eduardogmisiuk) [inline client/README.md:66]: to the device
in a dojot running

File: client/README.md (hunk 2 above)
Comment (mprevide) [reply client/README.md:41]: Actually is just one device 
`a device`
thx :)

**Review by eduardogmisiuk — APPROVED** [review]
LGTM!

**Pull request merged**